Import the service rules (#5)
DFE-Digital/ecf2: documentation/service-rules/grant-funding-summary.md

# Grant funding summary

## Grant funding for induction and training

There are 2 types of grant funding -- for induction and for training.

> -   Induction grant funding includes
> 
>     -   Year 1 Core Funding.
> 
>     -   Year 2 Time of Timetable Funding. Covers both:
> 
>         -   5% off timetable in year 2 of induction
> 
>         -   20 hours of mentor support in year 2 of induction
> 
> -   This funding is dedicated only for schools on the FIP programme.
>     Training grant funding includes:
> 
>     -   Year 1 & 2 Lead Provider Funding.
> 
>     -   Backfill mentor Funding.

Reviewer: ```suggestion
Induction grant funding includes:
    -   Year 1 Core funding

    -   Year 2 Time off timetable funding
    
Year 2 Time off timetable funding is comprised of:
-  5% off timetable in year 2 of induction
-  20 hours of mentor support in year 2 of induction

Training grant funding is dedicated only for schools on the FIP programme. This includes:

    -   Year 1 & 2 Lead Provider funding

    -   Backfill mentor funding
```

    Just FYI, a 'call-off contract' is (by definition) a contract
    between a supplier and a buyer. In this scenario, there is also a
    data point called 'call-off contract'

-   A call-off contract is by definition a contract between a supplier
    and a buyer. the call off contract is linked to a lead provider that
    a participant is associated with. This determines the funding 'pot'
    which pays for the participant's training. For example, 'a call off
    contract with EDT to train participants who began their training in
    the 2022--23 academic year'.

-   call_off_contract_year is an academic year within a call off
    contract. There can be multiple call_off_contract_years per
    call_off_contract. For example, the 2023 call-off contract covers
    participants who start in the 2023--24 academic year, and the
    2024--25 academic year. These are both call_off_contract_years. This
    is the true definition of the word 'cohort'.

-   Money is allocated per cohort. A cohort is a mirror of contract
    year, which follows an academic year. There is a call_of_contract
    per lead provider per the year. For example - for the Lead provider

>     \'Ambition\' there are 2 call_of_contract data points, one for the

Reviewer: ```suggestion
    'Ambition' there are 2 call_of_contract data points, one for the
```

# Grant funding summary

## Grant funding for induction and training

There are 2 types of grant funding -- for induction and for training.

Induction grant funding includes:

- Year 1 Core funding

- Year 2 Time off timetable funding
    
Year 2 Time off timetable funding is comprised of:

-  5% off timetable in year 2 of induction

-  20 hours of mentor support in year 2 of induction

Training grant funding is dedicated only for schools on the FIP programme. This includes:

- Year 1 & 2 Lead Provider funding

- Backfill mentor funding

## Training grant funding. Year 1 & 2 Lead Provider (LP)  Funding. 

### What does it consist of?

DfE pays Lead Providers to train ECTs and mentors in the full induction
programme. Lead Providers have to give proof that they are in fact
training mentors and ECTs, they do this through 'declarations'.
Declarations are statements of work. They are meant to provide 3

> declarations per year, 6 for both years. They do this through the API
> 'service' DfE supports.

Reviewer: ```suggestion
declarations per year, 6 for both years. They do this through the Lead provider API.
```

### What does it consist of?

DfE pays Lead Providers to train ECTs and mentors in the full induction
programme. Lead Providers have to give proof that they are in fact
training mentors and ECTs, they do this through 'declarations'.
Declarations are statements of work. They are meant to provide 3
declarations per year, 6 for both years. They do this through the API
'service' DfE supports.

### How are the payments structured?

To manage payments, DfE has ongoing contracts (also called 'call off
contracts') with LPs that cover the 2 years of training a group of ECTs
and mentors receive. Each academic year there is a new 'call off
contract' to cover the new group of participants that comes in. Each new
group of participants is often called a 'cohort'.

### So it's just a one-off payment?

No. There are 3 types of payments to LPs.

-   #### Service fees.
    These are fixed. They are an amount that
    providers will be paid regularly regardless of things like
    participant numbers. These are just a set payment for providing the
    service. They are paid monthly for a set time. Once passed,
    statements have £0 service fees.

-   #### Output fees. 

>     These are flexible. DfE pays providers an output
>     fee, whose value depends on a range of factors, when providers
>     train a participant up to a set milestone. Providers report this via
>     the API (subject to validation) using \'declarations\'.

Reviewer: ```suggestion
    These are flexible. When providers train a participant to a set milestone DfE pays them an output fee. The value of this fee depends on a range of factors. Providers report this via
    the API (subject to validation) using 'declarations'.
```

-   #### Uplift fees.

>     DfE pays an additional \'uplift fee\' for

Reviewer: ```suggestion
    DfE pays an additional 'uplift fee' for
```

    participants in specific courses and circumstances e.g. schools with
    [high pupil
    premium](https://www.gov.uk/government/publications/pupil-premium/pupil-premium)
    (pupils who are recorded as eligible for free school meals, looked
    after by a social worker, etc).

### What is the journey?

> Internal DfE staff can access providers\' financial statements via the
> finance tool. When a milestone period is hit, DfE staff conduct
> assurance and then send out a copy of the financial statement to Lead
> Providers, along with a CSV of the declarations that make up the
> statement. Lead Providers check this against what they expected and then
> invoice DfE. EFSA then makes the payment.\

Reviewer: ```suggestion
Internal DfE staff can access provider financial statements via the
finance tool. When a milestone period is hit, DfE staff conduct
assurance and then send out a copy of the financial statement to Lead
Providers, along with a CSV of the declarations that make up the
statement. Lead Providers check this against what they expected and then
invoice DfE. EFSA then makes the payment.
```

Throughout the process DfE also conducts audits, where they will select
a sample of LP data and ask for evidence of training, to reduce the risk
of any fraudulent payments.

### What is the eligibility policy?

All lead providers must declare their subcontracting arrangements (with
delivery partners) for the period 1 August to 31 July through the
[manage your education and skills funding
(MYESF)](https://skillsfunding.service.gov.uk/) service. Lead Providers
are required to report at least twice a year.

At the start of the period, Lead Providers must:

-   Submit their forecasted delivery on the number of ECTs and mentors
    trained

-   Update their declaration whenever they make any change to their
    Delivery Partners -- this includes adding or removing subcontractors
    (delivery partners) or changing the information of a previously
    submitted declaration.

At the end of the period, Lead Providers should submit their actual
delivery.

> If Lead Providers do not subcontract but receive direct funding, they

Reviewer: I'm not sure I fully understand this one

> If Lead Providers do not subcontract but receive direct funding, they

Reviewer 2: apologies, move sentence from line 102 and 103 
```suggestion
If Lead Providers do not subcontract but receive direct funding, they need to make a 'nil declaration'. 
```
to line 126

    to them.

-   Replacing mentor. A mentor who is already on the programme who
    starts to support an ECT (new or existing). This mentor does not
    need new funding as they have already had this allocated.

### How are the payments structured?

There are 2 payments made to schools at two different moments. If the
mentor has transferred, it is paid to the latest school the mentor has
been registered. This school is responsible for breaking up the grant
and dividing it between previous schools the mentor was at.

To release payment for year 1, the grant funding team needs to do the
following checks.

-   Make sure the mentor's lead provider has a 'start' declaration (the
    declaration that confirms the beginning of training).

-   Make sure the mentor cannot have withdrawn from the DfE-funded
    mentor training as of the date the data cut happens (1 June 2023).

-   An 'instalment 1' payment should not have been paid for that mentor
    previously as well (to prevent double funding).

To release payment for year 2, the grant funding team needs to do the
following checks.

-   Make sure the mentor's Lead Provider has submitted at least one

>     'engagement declaration' (either declaration 4, 5 or \'completion\')

Reviewer: ```suggestion
    'engagement declaration' (either declaration 4, 5 out of 6 or the last completion declaration)
```

>     'engagement declaration' (either declaration 4, 5 or \'completion\')

Reviewer: resolved in another suggestion to fix the /

    mentor can receive it if they meet the criteria

-   If the original mentor didn't receive instalment 2, the replacement
    mentor can receive it if they meet the criteria

-   If the replacement mentor did receive an instalment, the replacement
    mentor can also receive the same instalment if they are linked to a
    second ECT

### What is the journey for Beneficiaries?

1.  School induction tutors register mentors with DfE using manage
    training for ECTs 

2.  CPD sends each mentor an email (so it's important the addresses
    provided are for active email accounts) 

3.  CPD checks that the mentor:

    a.  does not have any prohibitions, sanctions or restrictions on
        their record

    b.  has not previously received funding for ECF based mentor
        training

4.  SIT logs into manage training to check the status of these
    eligibility checks and get confirmation the mentor is ready to go.

### What is the eligibility policy?

To be eligible as a school for this type of funding, you need to be

-   A state-funded school and establishment where induction can be
    served

-   Have a mentor signed up through manage training portal

-   Doing FIP programme

> For the mentor to be eligible they need to
> 
> -   Be on a FIP programme
> 
> -   Have an active email
> 
>   
> 
> Sign into manage training service and tell DfE their teacher reference
> number (TRN) or National Insurance number, name and date of birth
> 
> -   Not have any (serious) prohibitions, sanctions or restrictions on
>     their record
> 
> -   Can't have previously received funding for ECF based mentor training

Reviewer: ```suggestion
For the mentor to be eligible they need to

-   Be on a FIP programme

-   Have an active email

- Sign into manage training service and tell DfE their teacher reference
number (TRN) or National Insurance number, name and date of birth

-   Not have any (serious) prohibitions, sanctions or restrictions on
    their record

-   Can't have previously received funding for ECF based mentor training
```

## Induction grant funding. Year 1 Core Funding. 

This funding is meant to cover time off an ECT's school timetable to do
induction (10% of their scheduled time). It is provided by the
[Dedicated Schools Grant
(DSG)](https://www.gov.uk/government/collections/local-authorities-pre-16-schools-funding#dedicated-schools-grant),
calculated using the [national funding formula
(NFF)](https://www.gov.uk/government/publications/national-funding-formula-for-schools-and-high-needs).
The grant is paid to local authorities, who distribute the payment to
schools, by
[EFSA](https://www.gov.uk/government/organisations/education-and-skills-funding-agency)
(funding institution).

## Induction grant funding. Year 2 Time off Timetable.

### What does it consist of?

This funding is meant to cover time off an ECT's school timetable to do
induction (5% of their scheduled time) and a mentor's time off timetable
to support the ECT through induction (20 hours). This funding is for all
schools regardless of programme.

> To calculate this grant, the funding team need to know about training
> data, specifically, declarations. This is because our ways to track an
> ECTs induction time are messy, but the data provided for training
> declarations is as validated as it gets.

Reviewer: ```suggestion
To calculate this grant, the funding team need to know which ECTs and mentors are on their second year of induction. However our existing data model was not originally created with this task in mind. Hence we can't rely on induction data, we have to rely on training data, specifically, lead provider declarations as they are the 'cleanest' form of ECT data we have. 
```

### How are the payments structured?

The grant funding team need to firstly figure out which ECTs and mentor
and in their year 2 of their induction. This is a surprisingly difficult
thing to calculate as years doing induction is NOT the same as years
doing training. ECTs pause, stop, defer, change schools often and track
these changes is at the core of many issues in the data collection
system.

How 'Are they in their 2nd year?' is calculated.

-   For FIP -- if they have 4 declarations, we can safely assume they
    are in their second year of training.

-   For CIP -- CIP don't have declarations, so instead of trying to get
    who IS in the 2nd year, they calculate who is NOT -- who has NOT
    completed their first year based on their training registration
    date.

-   For DIY -- We have to check against DQT, SWO and call ABs directly
    and ask for ECT details.

Getting this data right is difficult because:

-   Tracking the movements of an ECT (pause, stop, defer, changing

>     schools\...) is dependent on AB and SIT updates. There are delays in

Reviewer: ```suggestion
    schools) is dependent on AB and SIT updates. There are delays in
```

    this data coming through.

-   Once the data does come through, the data gathered for each
    programme type is different.

> ### NEED TO CHECK THIS WITH GRANT FUNDING TEAM

Reviewer: What text from the doc? If you could make the change that would be handy, I can change you how. I don't know if it's added from the draft, this is all from the Service rules doc.

> ### NEED TO CHECK THIS WITH GRANT FUNDING TEAM

Reviewer 2: from line 267 substitute with:

### How are the payments structured? 

The grant funding team need to firstly figure out which ECTs and mentor and in their year 2 of their induction. This is a surprisingly difficult thing to calculate as years doing induction is NOT the same as years doing training. ECTs pause, stop, defer, change schools often and track these changes is at the core of many issues in the data collection system.    

### How 'Are they in their 2nd year?’ is calculated.  

- For FIP – if they have 4 declarations, we can safely assume they are in their second year of training.   
- 
- For CIP – CIP don’t have declarations, so instead of trying to get who IS in the 2nd year, they calculate who is NOT – who has NOT completed their first year based on their training registration date.   
- 
- For DIY – We have to check against DQT, SWO and call ABs directly and ask for ECT details.   

### Getting this data right is difficult because   

Tracking the movements of an ECT (pause, stop, defer, changing schools...) is dependent on AB and SIT updates. There are delays in this data coming through.   

Once the data does come through, the data gathered for each programme type is different.   

### What is the eligibility policy? 

From a policy standpoint, ECTs need to be the following to be eligible for Y2TOT funding.  
 

- Work in one of the schools and establishments serving induction 
- 
- Hold qualified teacher status (QTS) 
- 
- Be registered as being in their second year of their induction after 1 September 2021 

Schools and establishments who employ ECTs with qualified teacher learning and skills status (QTLS) are not eligible for DfE funding, because they are exempt from statutory induction. 

### 2024-2025 changes 

**Backfill mentor funding changes** 

Previously grant funding team were paying just on a start declaration at the beginning of the term and then the completed declaration at the end of year 2. However, the constant transfers, contract changes, AB changes, etc of mentors were causing a great deal of data errors (specifically 12% payment error, above the KPI mark). For this reason, from 2024 backfill mentor payments will be calculated pro rata based on the declarations for each mentor and will be paid to each school individually, instead of only the last school the mentor was placed at. For example - _Mentor A starts at School A and only does 1 term. Then Mentor A leaves, Mentor B takes over and does mentor training on School A for two more terms. Grant funding will pay School A the three terms in total. However, if Mentor A leaves and no one takes over, grant funding team would only pay the school 1/3 of that payment allowance. This data still deeply depends on the declaration data from LPs._  
Mentor training will be only 1 year. So backfill mentor grant funding payment will be less (number pending). BUT Y2TOT mentor hours will increase, so the ECT will have more mentor hour support.  

**Timing of payments** 

In 2024 the payments will be done in November instead of August to match up with the time LPs have to (under contract) finalize their declarations

-   Once the data does come through, the data gathered for each
    programme type is different.

> \-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-

Author: This looks a bit funny to me, should it be a regular horiztonal line `---`?

Reviewer: I think this might be funky formatting from the original word doc that got missed

> \-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-

Reviewer: ```suggestion
```
